web/bitwarden/bitwarden.go

```diff
@@ -44,11 +44,11 @@ func GetProfile(c echo.Context) error {
 			"error": "invalid token",
 		})
 	}
-	settings, err := settings.Get(inst)
+	setting, err := settings.Get(inst)
```

Comment:
How come the linter didn't error out on this reassignment of `settings`?

Reply:
I don't see a linter that checks that on https://github.com/golangci/awesome-go-linters